autograder/cli/common.py

```diff
@@ -1,5 +1,13 @@
-HEADERS = ['email', 'name', 'role', 'lms-id']
-SYNC_HEADERS = HEADERS + ['operation']
+USER_HEADERS = ['email', 'name']
+
+COURSE_USER_HEADERS = USER_HEADERS + ['role', 'lms-id']
+SYNC_HEADERS = COURSE_USER_HEADERS + ['operation']
+
+BASE_SERVER_USER_HEADERS = USER_HEADERS + ['role']
+SERVER_USER_HEADERS = BASE_SERVER_USER_HEADERS + ['courses']
+COURSE_INFO_HEADERS = ['id', 'name', 'role']
+
+INDENT = '    '
```

Comment:
Can you lexicographically re-order the constants , as that format is consistent throughout the repo